docs/source/archive/gsoc/reports/2024/scancode_toolkit_swastkk.rst

```diff
@@ -0,0 +1,69 @@
+========================================================================
+Compute summary for all detected packages.
+========================================================================
+
+
+| **Organization:** `AboutCode <https://aboutcode.org>`_
+| **Project:** `Scancode Toolkit <https://github.com/aboutcode-org/scancode-toolkit>`_
+| **Mentee:** `Swastik Sharma (swastkk) <https://github.com/swastkk>`_
+| **Mentors:** Philippe Ombredanne, AyanSinhaMahapatra, AvishrantSh, Jonathan Yang, Jay Kumar
+
+Overview
+--------
+
+Previously, we computed the summary at the codebase level, which included elements like the
+`license_clarity_score`, `declared_holder`, `other_license_expressions`, and more.
+This project aims to improve scanning accuracy by computing summaries and license clarity scores for
+each package and its files, rather than for the entire scan. This involves enhancing package models
+and ensuring accurate attribute collection across all package ecosystems.
+
+Implementation
+--------------
+
+All the work I did is contained in `this single PR <https://github.com/aboutcode-org/scancode-toolkit/pull/3792>`_.
```

Comment:
Make this a seperate section and create a list of PRs, it's possible we ask you to break the PRs in multiple parts or ask for seperate PRs.